x-pack/solutions/security/plugins/elastic_assistant/server/routes/register_routes.ts

```diff
@@ -103,6 +104,9 @@ export const registerRoutes = (
   getEvaluateRoute(router);
   postEvaluateRoute(router);
 
+  // Users
+  suggestUsersRoute(router, logger);
```

Comment:
Would be nice to update tests in the `x-pack/solutions/security/plugins/elastic_assistant/server/routes/register_routes.test.ts` to make sure this new endpoint won't be accidentally removed.